pendulum/pendulum.py

```diff
@@ -71,6 +71,10 @@ def _safe_create_datetime_zone(cls, obj):
             # pytz
             if hasattr(obj, 'localize'):
                 obj = obj.zone
+            elif hasattr(obj, 'utcoffset'):
```

Comment:
I think we can just do like in the `instance()` method and instead of raising an error just fallback on a `FixedTimezone` since we already know we have a `tzinfo` instance so `utcoffset()` is implied.